### PR TITLE
Adding inputs and outputs of CVM and HTM

### PR DESCRIPTION
**Pull request by mansisandag**
## Proposed changes

Documenting all input and output files of CVM Synthesis Establishment along with inputs of CVM and HTM

## Impact

Added the required CVM and HTM documentation

## Types of changes

What types of changes does your code introduce to ABM?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] Built the documentation locally and confirmed the updates.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: docs/inputs.md [hunk 1]
```diff
@@ -1260,6 +1260,30 @@ regional_pass: 10.90
...
+            Basic information of the data, and the size class definitions included
+        </td>
+    </tr>
+    <tr>
+        <td>Data</td>
+        <td>Detailed industry sector breakdown by size class and LUZ</td>
+    </tr>
+    
+</table>
+
```
Comment (JiaXu1024) [inline docs/inputs.md:1286]: @mansisandag , there are more inputs for CVM (see input/CVM). Please add the following files and fields descriptions:

- MGRAEmpByEstSize.csv
- SummarySynthEstabs.csv
- SynthEstablishments.csv

File: docs/inputs.md [hunk 2]
```diff
@@ -1707,6 +1731,159 @@ regional_pass: 10.90
...
+For exports, this is the US exit region where an export leaves United States.</td>
+    </tr>
+    <tr>
+        <td>Mode</td>
+        <td>Mode used between domestic origins and destinations</td>
+    </tr>
+    <tr>
+        <td>Commodity</td>
+        <td>2-digit level of the Standard Classification of Transported Goods (SCTG)
+</td>
```
Comment (JiaXu1024) [inline docs/inputs.md:1847]: Please double check from line #1833 to line#1847 to make sure format looks good.

File: docs/inputs.md [hunk 3]
```diff
@@ -1707,6 +1731,159 @@ regional_pass: 10.90
...
+For imports, this is the US entry region where an import enters United States.</td>
+    </tr>
+    <tr>
+        <td>dms_dest</td>
+        <td>FAF region or state where a freight movement ends the domestic portion of shipment. 
+For exports, this is the US exit region where an export leaves United States.</td>
+    </tr>
+    <tr>
+        <td>Mode</td>
+        <td>Mode used between domestic origins and destinations</td>
```
Comment (JiaXu1024) [inline docs/inputs.md:1842]: @mansisandag  Please add mode dictionary here.

File: docs/inputs.md [hunk 4]
```diff
@@ -1707,6 +1731,159 @@ regional_pass: 10.90
...
+    </tr>
+    <tr>
+        <td>Commodity</td>
+        <td>2-digit level of the Standard Classification of Transported Goods (SCTG)
```
Comment (JiaXu1024) [inline docs/inputs.md:1846]: @mansisandag Please add commodity groups dictionary here

File: docs/outputs.md [hunk 5]
```diff
@@ -3372,6 +3372,166 @@ This directory contains outputs from the Crossborder model, which represents all
...
+  </td>
+  </tr>
+  <tr>
+    <td>Size_Class
+    </td>
+    <td>The categorized size of the establishment based on the number of employees
+  </td>
+  </tr>
+</table>
+
```
Comment (JiaXu1024) [inline docs/outputs.md:3534]: @mansisandag Please modify from line #3375 to line#3533. Those files are inputs for CVM model (see input/CVM). Please move those to input page.

File: docs/outputs.md [hunk 6]
```diff
@@ -4578,6 +4738,281 @@ This directory contains San Diego commercial travel model outputs.
...
+    <td>Trip toll cost, from traffic_skims_MD
+    </td>
+  </tr>
+  <tr>
+    <td>costOperatingDrive
+    </td>
+    <td>Trip operating costs (cents)
+    </td>
+  </tr>
+</table>
```
Comment (JiaXu1024) [inline docs/outputs.md:5015]: @mansisandag see output/CVM, there are more output files generated from CVM. Please add those:

- final_commercial_accessbility.csv
- final_cv_trips.csv
- final_establishments.csv
- final_establishments_all.csv
- final_households.csv
- final_persons.csv
- final_land_use.csv

**Comment by mansisandag** [comment]
Issue with file changed ( reflected on github )

**Pull request closed without merging**